app/scripts/modules/ecs/serverGroup/configure/wizard/advancedSettings/advancedSettings.component.html

```diff
@@ -23,15 +23,13 @@
       <b>Placement strategy</b> <help-field key="ecs.placementStrategy"></help-field>
     </div>
     <div class="col-md-7">
-      <ui-select multiple
-                 ng-model="$ctrl.command.placementStrategy"
-                 class="form-control input-sm">
-        <ui-select-match>{{$item}}</ui-select-match>
-        <ui-select-choices repeat="item in ['AZ Balanced Spread', 'AZ Balanced BinPack', 'BinPack', 'One Task Per Host']">
+      <ui-select ng-model="$ctrl.command.placementStrategyName" class="form-control input-sm">
```

Comment:
please add a help pop-up that says that custom strategies are not yet implemented and PRs are welcome

Reply:
Done.